src/RSP.cpp

```diff
@@ -303,7 +303,7 @@ void RSP_Init()
 		config.generalEmulation.hacks |= hack_texrect_shade_alpha;
 	else if (strstr(RSP.romname, (const char *)"THE LEGEND OF ZELDA") != nullptr ||
 			 strstr(RSP.romname, (const char *)"ZELDA MASTER QUEST") != nullptr)
-		 config.generalEmulation.hacks |= hack_subscreen | hack_ZeldaMonochrome;
+		config.generalEmulation.hacks |= hack_subscreen | hack_ZeldaMonochrome;
```

Comment:
What is changed in this line?

Reply:
Looks like a formatting fix.

Reply:
Yeah it was just an extra space I spotted

Reply:
Ok